samples/samples/snippets.py

```diff
@@ -1407,6 +1407,29 @@ def insert_singers(transaction):
 # [END spanner_get_commit_stats]
 
 
+# [START spanner_set_max_commit_delay]
+def set_max_commit_delay(instance_id, database_id):
+    """Inserts sample data and sets a max commit delay."""
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+    database = instance.database(database_id)
+
+    def insert_singers(transaction):
+        row_ct = transaction.execute_update(
+            "INSERT Singers (SingerId, FirstName, LastName) "
+            " VALUES (110, 'Virginia', 'Watson')"
+        )
+
+        print("{} record(s) inserted.".format(row_ct))
+
+    database.run_in_transaction(
```

Comment:
Looks like parentheses not closed properly.

Reply:
Done.